### ISSUE TITLE
Documentation is hard to follow

### ISSUE THREAD

**Issue by arokem** [nipy/nibotmi#19]
Sorry - I don't understand the terminology. What are `slaves`? These are presumably the same thing as `buildslaves`? What is a `buildmaster`? How does that differ from `buildbot`? 

A gentle introduction to the system upfront would be nice. Or maybe a reference to another place that explains all these terms, and the structure of the system.

**Comment by matthew-brett** [comment]
I updated the docs a little.
